tests/mpi/renumber_cuthill_mckee.cc

```diff
@@ -22,6 +22,7 @@
 #include "../tests.h"
 #include <deal.II/base/logstream.h>
 #include <deal.II/base/mpi.h>
+#include <deal.II/base/mpi.templates.h>
```

Comment:
I see. That may not be the most elegant solution (using internal stuff in a test) but it's not worth worrying about I guess. OK with me then.